### DISCUSSION TITLE
`TypeError: __vite_ssr_import_0__ is not a function` on dayjs()

### DISCUSSION THREAD

**Discussion by zhengbangbo** [vitest-dev/vitest#2484] [Q&A] [answered]
I am switching [this project](https://github.com/zhengbangbo/component-library-list) from Jest to Vitest. 

The tests for `dayjs()` were running fine in Jest, but I don't know why they are giving this error in Vitest: 
```
TypeError: __vite_ssr_import_0__ is not a function
```

- [Commit](https://github.com/zhengbangbo/component-library-list/commit/d0d4adf514d0ef4aff9eeb55bc02d0556b89ef3d)
- [GitHub Action Job](https://github.com/zhengbangbo/component-library-list/actions/runs/3668349789/jobs/6201413223)

This is a small project and don't use Vite.

I found that all the other discussions were mock-related, and I didn't use mock. 
- https://github.com/vitest-dev/vitest/discussions/1255
- https://github.com/vitest-dev/vitest/discussions/619

The relevant part is about the date or time, I used [dayjs](https://github.com/iamkun/dayjs/).

**Comment by sheremet-va** [comment, marked as answer]
This should answer your question: https://github.com/vitest-dev/vitest/discussions/1993

TLDR:

Use `import dayjs from 'dayjs` instead of `import * as dayjs`
